src/kem/kyber/arithmetic.rs

```diff
@@ -17,7 +27,7 @@ const BARRETT_MULTIPLIER: i64 = 20159; // floor((BARRETT_R / FIELD_MODULUS) + 0.
 
 #[cfg_attr(hax, hax_lib_macros::requires((i64::from(value) > -BARRETT_R && i64::from(value) < BARRETT_R)))]
 #[cfg_attr(hax, hax_lib_macros::ensures(|result| result > -FIELD_MODULUS && result < FIELD_MODULUS))]
-pub(crate) fn barrett_reduce(value: KyberFieldElement) -> KyberFieldElement {
```

Comment:
Should we generally be removing all references to Kyber and replacing them with ML-KEM-based nomenclature?

Reply:
Yeah, might as well, unless there's another name change on the horizon ...